### PR TITLE
Fix send-mail error

### PR DESCRIPTION
**Pull request by adi88d**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: [link to the issue](https://jira-hq.paloaltonetworks.local/browse/XSUP-23910)
fixes: [link to the issue](https://jira-hq.paloaltonetworks.local/browse/XSUP-23911)

## Description
- Fixed and issue where **send-mail** command failed with the error:
`Timeout Error: Docker code script failed due to timeout, consider changing timeout value for this automation,
Error: Failed to decode (loop) data from docker code script: unexpected EOF`
- Added predefined values for `renderBody` argument

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

**Review by kobymeir — APPROVED** [review]

**Review by tkatzir — APPROVED** [review]

File: Packs/Gmail/Integrations/Gmail/Gmail.py [hunk 1]
```diff
@@ -2161,11 +2161,11 @@ def mail_command(args, subject_prefix='', in_reply_to=None, references=None):
     send_mail_result = sent_mail_to_entry('Email sent:', [result], email_to, email_from, cc, bcc, rendering_body,
                                           subject)
     if render_body:
-        html_result = CommandResults(
-            entry_type=EntryType.NOTE,
-            content_format=EntryFormat.HTML,
-            raw_response=html_body,
-        )
+        html_result = {
```
Comment (kobymeir) [inline Packs/Gmail/Integrations/Gmail/Gmail.py:2164]: @adi88d why revert back to raw json? this has not change implication in observing the command result translation to json.

Comment (adi88d) [reply Packs/Gmail/Integrations/Gmail/Gmail.py:2164]: I didnt revert back , just change the entry from CommandResults to html entry

Comment (kobymeir) [reply Packs/Gmail/Integrations/Gmail/Gmail.py:2164]: @adi88d what was there before was already an html entry. returned via CommandResult. there is no need for a change here.

Comment (adi88d) [reply Packs/Gmail/Integrations/Gmail/Gmail.py]: Thanks @kobymeir ! i revert this part

**Comment by adi88d** [comment]
Force merged due to failing TPB's

**Pull request merged**